### PR TITLE
Add CGP-37, Contracts Release 5

### PR DESCRIPTION
**Pull request by m-chrzan**

### CODE REVIEW

File: CGPs/0037.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- Status: DRAFT
+- Governance Proposal ID #: TBD
+- Date Executed: TBD
+- Discussions To: TBD
+
+## Overview
+
+In this governance proposal, we propose to execute the Celo Core Contracts Upgrade Release 5. This release follows the [release process](https://docs.celo.org/community/release-process/smart-contracts).
+
+Changes that are being proposed in this release are described in [this Github release](https://github.com/celo-org/celo-monorepo/releases/tag/core-contracts.v5.pre-audit).
```
Comment (yorhodes) [inline CGPs/0037.md:14]:
```suggestion
Changes that are being proposed in this release are described in [this Github release](https://github.com/celo-org/celo-monorepo/releases/tag/core-contracts.v5).
```

File: CGPs/0037.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Risks
+
+Celo Core Contracts are critical for the functioning of the Celo Platform. This proposal touches contracts which impact Celo's Proof-of-Stake and consensus and the stability protocol.
+While cLabs have invested significant efforts to expand testing and verification tooling, and a third party audit has been completed, errors in this proposal could lead to situations that are only recoverable with a very difficult hard fork.
+
+## Useful Links
+
+* [Celo Core Contracts Release Process](https://docs.celo.org/community/release-process/smart-contracts)
+* [Github release](https://github.com/celo-org/celo-monorepo/releases/tag/core-contracts.v5.pre-audit)
```
Comment (yorhodes) [inline CGPs/0037.md:48]:
```suggestion
* [Github release](https://github.com/celo-org/celo-monorepo/releases/tag/core-contracts.v5)
```

**Review by yorhodes — APPROVED** [review]

**Review by YazzyYaz — CHANGES_REQUESTED** [review]
Please submit to the new governance repo. 

CGPs are being removed from this repo. see https://github.com/celo-org/celo-proposals/pull/264

**Comment by yorhodes** [comment]
https://github.com/celo-org/governance/pull/16 @YazzyYaz

**Pull request closed without merging**